chapter00-valeurs-propres.tex

```diff
@@ -27,11 +27,11 @@ \section{Valeurs propres et vecteurs propres}
          & & λ_n
     \end{pmatrix},
   \end{displaymath}
-si et seulement si $v_i$ est un vecteur propre associé à la valeur propre $λ_i$ pour tout $i=1,\dots,n$.
+  si et seulement si $v_i$ est un vecteur propre associé à la valeur propre $λ_i$ pour tout ${i\in\{1,\dots,n\}}$.
 \end{lemma}
 
 \begin{proof}
-    Pour $v ∈V$ soit $[v]_B ∈K^n$ le vecteur des coordonnées de $v$ relatif à $B$. On a $[f(v_i)]_B = A \,  [v_i]_B$ pour $i=1,\dots,n$. Des que $[v_i]_B = e_i$, et $f(v_i) = λ_i v_i$ alors
+Pour $v ∈V$ soit $[v]_B ∈K^n$ le vecteur des coordonnées de $v$ relatif à $B$. On a $[f(v_i)]_B = A \,  [v_i]_B$ pour $i\in\{1,\dots,n\}$. Si $[v_i]_B = e_i$, et $f(v_i) = λ_i v_i$ alors
```

Comment:
```suggestion
Pour $v ∈V$ soit $[v]_B ∈K^n$ le vecteur des coordonnées de $v$ relatif à $B$. On a $[f(v_i)]_B = A \,  [v_i]_B$ pour $i\in\{1,\dots,n\}$. Puisque $[v_i]_B = e_i$, et $f(v_i) = λ_i v_i$ alors
```

I've made a mistake on this line, it should be "puisque" and not "si". Generally "des que" means "from the moment in time" rather than having a causal function in the sentence.